libxrdp/xrdp_caps.c

```diff
@@ -659,6 +671,9 @@ xrdp_caps_process_confirm_active(struct xrdp_rdp *self, struct stream *s)
             case 0x001E: /* CAPSSETTYPE_FRAME_ACKNOWLEDGE */
                 xrdp_caps_process_frame_ack(self, s, len);
                 break;
+            case RDP_CAPSET_SURFCMDS: /* CAPSETTYPE_SURFACE_COMMANDS */
```

Comment:
Why not just use Microsoft's names? https://msdn.microsoft.com/en-us/library/cc240486.aspx